sonic_platform_base/sonic_xcvr/api/public/cmis.py

```diff
@@ -367,6 +367,13 @@ def get_voltage(self):
             return None
         return float("{:.3f}".format(voltage))
 
+    def is_copper(self):
+        '''
+        Returns True if the module is copper, False otherwise
+        '''
+        media_intf = self.get_module_media_type()
+        return media_intf == "passive_copper_media_interface" or media_intf == "active_cable_media_interface"
```

Comment:
@active cable is not passive media. Active cables like AEC have datapath state machine. @mihirpat1 can you check

Reply:
@prgeor Confirmed that active cables like AEC have datapath state machine. Hence, this check needs to be modified.